### DISCUSSION TITLE
Unable to style nav link on hover

### DISCUSSION THREAD

**Discussion by noneall** [microsoft/fluentui#16206] [Q&A] [answered]
I try to style the anchor tag on hover in Nav component but its impossible to override the default style. [INavStyles ](https://developer.microsoft.com/en-us/fluentui#/controls/web/nav#INavStyleProps) does not have any prop to support hover case.

> `export const navStyle: Partial<INavStyles> = {
  chevronButton: {
    color: theme.palette.white,
  },
  link: [
    {
      color: theme.palette.white,
    },
    'nav-link',
  ],
  compositeLink: {
    '&:hover': {
      color: 'red',
      '.nav-link': {
        backgroundColor: 'blue',
      },
    },
  },
}
`
![image](https://user-images.githubusercontent.com/19172729/101875789-41fa8f80-3b59-11eb-92b0-c5f1fede69f8.png)
#office-ui-fabric, #fluentui-react, #fluentui

**Comment by ecraig12345** [comment, marked as answer]
You can do this using a `.ms-Nav-compositeLink:hover` selector within `styles.link`:

```ts
const styles: Partial<INavStyles> = {
  link: {
    '.ms-Nav-compositeLink:hover &': {
      backgroundColor: 'purple' // your real styling here
    }
  }
};
```

Here's a live demo: https://codepen.io/ecraig12345/pen/abZjZme
